auto_submit/lib/service/config.dart

```diff
@@ -37,6 +37,14 @@ class Config {
   static const String kAutosubmitLabel = 'autosubmit';
   static const String kRevertLabel = 'revert';
 
+  static final Set<String> approvedAuthorAssociations = <String>{'MEMBER', 'OWNER'};
+
+  // Regex for merge update comments
+  static final RegExp regExpMergeMethod = RegExp(r'@autosubmit\s*:\s*merge', caseSensitive: false);
```

Comment:
Are we having a similar comment for a rebase only?

Reply:
I don't believe you can select a rebase. It only has one endpoint in the api for updating a branch.
https://docs.github.com/en/rest/pulls/pulls?apiVersion=2022-11-28#update-a-pull-request-branch